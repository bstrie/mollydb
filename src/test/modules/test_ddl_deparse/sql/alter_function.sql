--
-- ALTER_FUNCTION
--

ALTER FUNCTION plmdb_function_trigger_1 ()
   SET SCHEMA foo;

ALTER FUNCTION foo.plmdb_function_trigger_1()
  COST 10;

CREATE ROLE tmprole;

ALTER FUNCTION plmdb_function_trigger_2()
  OWNER TO tmprole;

DROP OWNED BY tmprole;
DROP ROLE tmprole;
