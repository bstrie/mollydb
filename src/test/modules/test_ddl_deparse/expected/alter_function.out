--
-- ALTER_FUNCTION
--
ALTER FUNCTION plmdb_function_trigger_1 ()
   SET SCHEMA foo;
NOTICE:  DDL test: type simple, tag ALTER FUNCTION
ALTER FUNCTION foo.plmdb_function_trigger_1()
  COST 10;
NOTICE:  DDL test: type simple, tag ALTER FUNCTION
CREATE ROLE tmprole;
ALTER FUNCTION plmdb_function_trigger_2()
  OWNER TO tmprole;
ERROR:  function plmdb_function_trigger_2() does not exist
DROP OWNED BY tmprole;
DROP ROLE tmprole;
